kratos/utilities/normal_calculation_utils.cpp

```diff
@@ -24,7 +24,9 @@
 namespace Kratos
 {
 template <class TContainerType>
-void NormalCalculationUtils::CalculateNormalsInContainer(ModelPart& rModelPart)
+void NormalCalculationUtils::CalculateNormalsInContainer(
+    ModelPart& rModelPart,
+    const Variable<array_1d<double,3>>& rNormalVariable)
```

Comment:
```suggestion
    const Variable<array_1d<double,3>>& rNormalVariable
    )
```

Reply:
Why all these changes? I've just checked our [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide) and there is no comment about the brackets.

Reply:
Previous code was like that (see diff)